README.md

```diff
@@ -3254,6 +3256,7 @@ _Software written in Go._
 - [trubka](https://github.com/xitonix/trubka) - A CLI tool to manage and troubleshoot Apache Kafka clusters with the ability of generically publishing/consuming protocol buffer and plain text events to/from Kafka.
 - [uTask](https://github.com/ovh/utask) - Automation engine that models and executes business processes declared in yaml.
 - [Vegeta](https://github.com/tsenart/vegeta) - HTTP load testing tool and library. It's over 9000!
+- [version-manager](https://github.com/gvcgo/version-manager) - A general version manager for multiple SDKs, such as Go, Java, Node.js, Python, .Net, etc.
```

Comment:
https://github.com/gvcgo/resources seems to be the underlying version tracker. Which you are updating using your account automation. With a lot of empty commits https://github.com/gvcgo/resources/commits/main/
- https://github.com/gvcgo/resources/commit/c1a4854ffb9e83f903469490b44d640f233889c8
- https://github.com/gvcgo/resources/commit/25eae64634beb5a06e1608a2ea41432cf42766c9
- https://github.com/gvcgo/resources/commit/1a5f08222dc16e1c4825c086b98cf598d635d429
This is a supply chain attack waiting to happen, you are not even using github actions so people can see what you could commit based on the action logic. I will not be accepting this either.
And also you are packaging your go fork of asciinema with this instead of the official one.